data/src/main/kotlin/io/timemates/backend/data/files/datasource/FileEntityMapper.kt

```diff
@@ -11,12 +11,12 @@ class FileEntityMapper {
     fun toDomainFile(
         fileId: String,
         fileType: FileType,
-        imageSize: Int?
+        imageSize: Int
```

Comment:
trailing comma after last argument (then it's much easier to define who made changes, new argument will lead to situation when it shows that another user made change regardless you did it)